### DISCUSSION TITLE
Changing link in navigation bar

### DISCUSSION THREAD

**Discussion by ahmedivy** [withastro/starlight#2020] [Q&A] [answered]
I am using starlight for documentation and starlight blog plugin for blog, when on /docs page, there is a link to /blog page on top of navigation bar, but when i go to blog site, this link stays same, i want so that when it is on /blog page, it show the link back to /docs page. I wasn't able to find any customization option for this, please help me if thats do able. Thank you.

<img width="369" alt="image" src="https://github.com/withastro/starlight/assets/91587113/c6e515d4-6c1c-42a1-b73c-74fd4e974856">

Content Config:
```ts
import { docsSchema } from "@astrojs/starlight/schema";
import { defineCollection } from "astro:content";
import { blogSchema } from "starlight-blog/schema";

export const collections = {
  docs: defineCollection({ schema: docsSchema({ extend: blogSchema() }) }),
};
```

Astro Config:
```ts
import tailwind from "@astrojs/tailwind";
import { defineConfig } from "astro/config";
import starlightBlog from "starlight-blog";

import starlight from "@astrojs/starlight";

// https://astro.build/config
export default defineConfig({
  integrations: [
    tailwind(),
    starlight({
      title: "SaltLakeCity.dental",
      customCss: [
        "./src/styles/docs.css",
        "@fontsource-variable/mada",
        "@fontsource-variable/montserrat",
        "@fontsource/palanquin/300.css",
        "@fontsource/palanquin/400.css",
        "@fontsource/palanquin/500.css",
        "@fontsource/palanquin/600.css",
      ],
      plugins: [starlightBlog({})],
      sidebar: [
        {
          label: "Docs",
          autogenerate: { directory: "docs" },
        },
      ],
    }),
  ],
});
```

**Comment by HiDeoo** [comment, marked as answer]
The question is more related to the blog plugin but you should be able to provide your own [override](https://starlight.astro.build/guides/overriding-components/) for the [`ThemeSelect`](https://starlight.astro.build/reference/overrides/#themeselect) component.

This is the component overriden by the blog plugin to add the link you are referring to and by providing your own override you would be able to have control over the link.

Just like the [blog plugin override](https://github.com/HiDeoo/starlight-blog/blob/main/packages/starlight-blog/overrides/ThemeSelect.astro), you own override will need to [reuse](https://starlight.astro.build/guides/overriding-components/#reuse-a-built-in-component) the built-in `ThemeSelect` component after adding your own custom link which would change [based on the current route](https://starlight.astro.build/guides/overriding-components/#use-page-data).

An example of the override could look like this:

```jsx
---
import Default from '@astrojs/starlight/components/ThemeSelect.astro'
import type { Props } from '@astrojs/starlight/props'

const isBlog = Astro.props.slug.startsWith('blog')
---

<div>
  <a href={isBlog ? '/' : '/blog/'}>{isBlog ? 'Docs' : 'Blog'}</a>
</div>
<Default {...Astro.props}><slot /></Default>

<style>
  div {
    border-inline-end: 1px solid var(--sl-color-gray-5);
    display: none;
    padding-inline-end: 1rem;
  }

  @media (min-width: 50rem) {
    div {
      display: flex;
    }
  }

  a {
    color: var(--sl-color-text-accent);
    font-weight: 600;
    text-decoration: none;
  }
</style>
```

**Reply by ahmedivy** [reply]
It worked. Thank you so much for quick reply
